docs/_includes/calendar.html

```diff
@@ -93,15 +93,15 @@ <h3>Calendar with Disabled Cells and "Today"</h3>
                 <tbody>
                     <tr>
                         <td colspan="4"></td>
-                        <td><span class="calendar__cell--disabled">1</span></td>
-                        <td><span class="calendar__cell--disabled">2</span></td>
-                        <td><span class="calendar__cell--disabled">3</span></td>
+                        <td><span aria-disabled="true">1</span></td>
```

Comment:
This needs to use clipped text instead of aria-disabled. It should convey the meaning of the color, e.g. "past date".